_data/tech.yml

```diff
@@ -244,6 +244,16 @@ companies:
     date-updated: YYYY
     status: No
 
+  - name: Shutterstock
+    url: https://www.shutterstock.com/
+    twitter: Shutterstock
+    img: shutterstock.jpg
+    diversitydata: No
+    eeo-1: No
+    documentation: <link to company's diversity data>
```

Comment:
I would mention that this could be an empty value but there's others like this so I'll just clean that up after this gets merged.